src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -212,21 +212,10 @@ void Config::UpdateFromMap(const std::map<std::string, std::string>& config) {
                 THROW_GNA_EXCEPTION << "GNA pwl uniform algorithm parameter "
                                     << "should be equal to YES/NO, but not" << value;
             }
-        } else if (key == GNA_CONFIG_KEY(PWL_MAX_ERROR_PERCENT) || key == ov::intel_gna::pwl_max_error_percent) {
-            float max_error;
-            try {
-                max_error = InferenceEngine::CNNLayer::ie_parse_float(value);
-                if (max_error < 0.0f || max_error > 100.0f) {
-                    throw std::out_of_range("");
-                }
-            } catch (std::invalid_argument&) {
-                THROW_GNA_EXCEPTION << "Invalid value of PWL max error percent";
-            } catch (std::out_of_range&) {
-                THROW_GNA_EXCEPTION << "Unsupported PWL error percent value: " << value
-                                    << ", should be greater than 0 and less than 100";
-            }
-            gnaFlags.pwlMaxErrorPercent = max_error;
-            OPENVINO_SUPPRESS_DEPRECATED_END
+        } else if (key == GNA_CONFIG_KEY(PWL_APPROXIMATION_MODE) || key == ov::intel_gna::pwl_approximation_mode) {
+            ov::intel_gna::PWLApproximationMode pwl_approximation_mode =
```

Comment:
any reason to use this proxy variable?

Reply:
It left after after refactoring, will fix it.